src/k8s-engine/notimplemented.js

```diff
@@ -128,6 +128,18 @@ export class OSNotImplemented extends events.EventEmitter {
 
     return Promise.reject(new Error('not implemented'));
   }
+
+  getFailureDetails() {
+    this.#notified = displayError(this.#notified);
+
+    return Promise.reject(new Error('not implemented'));
+  }
+
+  get lastCommandComment() {
```

Comment:
This technically doesn't implement the interface (we only care about the setter, and it's missing).

Reply:
Think this is done now.